File: c++/calculate.cpp
#include<bits/stdc++.h>
using namespace std;


// inline double f(double x)
// {
//     return 1.0/((x-1)*(x-1));
// }
inline double f(double x,double y)
{
    return 2*x*y+2*y;
}
inline double y(double x)
{
    return exp(x*x+2*x+log(3));
}
int main()
{
    // while(x+dx<=b+e)
    // {
    //     sum+=dx*.5*(f(x+dx)+f(x));
    //     x=x+dx;
    // }
    // printf("%lf\n",sum);
    // x=a,sum=0;
    // while(x+2*dx<=b+e)
    // {
    //     sum+=(dx/3)*(f(x)+4*f(x+dx)+f(x+2*dx));
    //     x+=2*dx;
    // }
    // printf("%lf",sum);
    double dx=1e-3;
    double x=0,y1=3;

    int n=10000000;
    while(n--)
    {
        y1=y1+f(x,y1)*dx;
        x+=dx;
        printf( "%lf %lf %lf %lf\n",x,y1,y(x),y(x)-y1);

    }
}

File: python/calculate.py
import math 
# import flask
# import latex2sympy2
# e=1e-5
# x=e
# sum=0
# while x<1-e:
#     sum+=e*(-math.log(1-x)/x)
#     x+=e
# print(sum-math.pi**2/6)
ans=0
for i in range(15):
    ans+=5**i*math.exp(-5)/math.factorial(i)
print(ans)
print(format(1-ans,'.4e'))